packages/web/src/components/nav/mobile/TopLevelPage.tsx

```diff
@@ -18,15 +17,12 @@ const rootElement =
   typeof document !== 'undefined' ? document.querySelector('#root') : null
 
 const TopLevelPage = ({ showAddToCollection }: TopLevelPageProps) => {
-  const { isOpen } = useEditPlaylistModal()
-  const showPage = isOpen || showAddToCollection
+  const showPage = showAddToCollection
```

Comment:
nit: could remove this var and use `showAddToCollection` below

Reply:
will dooo